### PR TITLE
Create adapt-to-large-screens

### PR DESCRIPTION
**Pull request by mariamhas**
_Description of what this PR is changing or adding, and why: Adding guidelines and best practices for developing flutter apps for large screen device formats.

_Issues fixed by this PR (if any):https://github.com/flutter/website/issues/9896

## Presubmit checklist

- [ x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [ x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x ] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/ui/layout/responsive/adapt-to-large-screens.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+## Why does this matter for developers?
+
+When your app supports large screens, they will also receive the
+following benefits.
+
+* Increases your business metrics. They would show increased user 
+  engagement on tablets for apps with layouts optimized to fill the full screen.
+* Improves the [technical quality][] to increase visibility for your
+  app in the Play Store on these devices. Recent [Play Store updates][] show 
+  ratings by device type and notify users when an app lacks large screen support.
```
Comment (sfshaza2) [inline src/ui/layout/responsive/adapt-to-large-screens.md:24]: Are there benefits to iOS apps, as well?

```suggestion
When your app supports large screens, it also receives the
following benefits:

* Improves your app's metrics to show increased user engagement
  when optimized to fill the screen.
* Increases your app's visibility in the Play Store.
  Recent [Play Store updates][] show ratings by device type and
  indicates when an app lacks large screen support. 
```

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
@mariamhas, I have some rather large edits I'd like to see for this page. Perhaps we can hash it out in a Google doc? My comments include a fairly significant reorganization. I'm happy to meet via GVC if that helps, though I do have next week off.

**Comment by mariamhas** [comment]
@sfshaza2 thanks for reviewing shams! I added you to the doc where I had originally written this pr. There is no rush, happy to meet in the new year to discuss changes!

**Comment by sfshaza2** [comment]
I'm closing this, because I'm working on updating that section. But I WILL use some of your great content, @mariamhas!

**Pull request closed without merging**